uaclient/config.py

```diff
@@ -670,11 +670,13 @@ def _unattached_status(self) -> Dict[str, Any]:
 
             response["services"].append(
                 {
-                    "name": resource["name"],
+                    "name": resource.get("presentAs", resource["name"]),
                     "description": ent_cls.description,
                     "available": available,
                 }
             )
+        response["services"].sort(key=lambda x: x.get("name", ""))
```

Comment:
Is there a reason to perform the sort here instead of the current approach ?

Reply:
Yes - CIS and USG appear in different order and we don't know which one will be there before looping.
If we sort first by name, and then the `presentAs` is different, it ends up in the wrong position.